tests/Sniffs/Attributes/AttributesOrderSniffTest.php

```diff
@@ -34,6 +34,7 @@ public function testErrors(): void
 				'Group\\Attribute2',
 				'Group\\',
 			],
+			'orderAlphabetically' => true,
```

Comment:
Please create two tests for this option: `noErrors` and `errors`

Reply:
Done